conformance/scenarios/httpgateway/httpgateway_test.go

```diff
@@ -0,0 +1,141 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package httpgateway
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"sigs.k8s.io/gateway-api/apis/v1alpha1"
+	"sigs.k8s.io/gateway-api/conformance/utils"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/klog"
+)
+
+func TestHttpGateway(t *testing.T) {
+	// Test Setup
+	var err error
+	utils.KubeClient, utils.GWClient, err = utils.LoadClientset()
+	if err != nil {
+		klog.Fatalf("error loading Kubernetes clientsets: %v", err)
+	}
+	err = utils.DynamicApply(utils.DynamicParams{Path: "../../../config/crd/bases"})
+	if err != nil {
+		t.Fatalf("error installing gateway-api CRDs: %v", err)
+	}
+	ns, err := utils.NewNamespace()
+	if err != nil {
+		t.Fatalf("error creating namespace: %v", err)
+	}
+
+	dp := utils.DynamicParams{
+		Path:      "httpgateway.yaml",
+		Namespace: ns,
+	}
+	err = utils.DynamicApply(dp)
+	if err != nil {
+		t.Fatalf("error installing gateway-api examples: %v", err)
+	}
+
+	t.Run("Gateway.Status.Conditions Scheduled condition should be set to true within 3 minutes", func(t *testing.T) {
```

Comment:
nit: 3 minutes in description, but its 1 minute in practice

Reply:
Good catch, was a temporary workaround for the sake of debugging that I forgot to revert.